src/mcts/search.cc

```diff
@@ -1216,11 +1216,41 @@ void SearchWorker::GatherMinibatch() {
     if (search_->stop_.load(std::memory_order_acquire)) return;
   }
 }
+
+namespace {
+int Mix(int high, int low, float ratio) {
```

Comment:
fyi there will be std::lerp soon: <https://en.cppreference.com/w/cpp/numeric/lerp>
But that's not exactly the same as here also rounding comes.

Also `low + ratio * (high-low)` may be more more readable (and a tiny bit faster because of one multiplication instead of two).

Reply:
I'll make the change, on simplification grounds rather than readable grounds.